### PR TITLE
Update measures due to BNF code changes

### PR DESCRIPTION
**Pull request by richiecroker**
amending for BNF changes - makes classification easier

### CODE REVIEW

**Review by a-d-brown — COMMENTED** [review]
Looks good to me. The only thing I notice is that in the VSL/Vivomixx measure the liquid OTC code captures some additional things i.e. 'Yakult' and 'Symprove'. Is that intentional? If so, I wonder if we should re-title the measure as something more general like 'Non-ACBS Probiotics'?

**Review by a-d-brown — APPROVED** [review]

File: openprescribing/measure_definitions/aafpercent.json [hunk 1]
```diff
@@ -1,56 +1,49 @@
...
+	],
+	"denominator_type": "bnf_quantity",
+	"denominator_bnf_codes_filter": [
+		"0913107A0 # Powder amino acid formula (0913107)",
+		"0913131A0 # Powder amino acid formula (0913131)",
+		"0913131B0 # Powder amino acid semi solid  (0913131)",
+		"0913355A0 # Powder discontinued amino acid formula (0913355)",
+		"0913361A0 # Powder discontinued amino acid formula (0913361)",
+		"0913101A0 # Powder extensively hydrolysed formula (0913101)",
+		"0913101A0 # Powder extensively hydrolysed formula (0913101)",
```
Comment (chrisjwood16) [inline openprescribing/measure_definitions/aafpercent.json:38]: This line is repeated

Comment (richiecroker) [reply openprescribing/measure_definitions/aafpercent.json]: good point, have removed!

File: openprescribing/measure_definitions/cmpaperpt.json [hunk 2]
```diff
@@ -21,35 +17,26 @@
...
-    "FROM {measures}.cmpa_products",
-    "WHERE type != 'exclude'"
+  "numerator_bnf_codes_filter": [
+		"0913107A0 # Powder amino acid formula (0913107)",
+		"0913131A0 # Powder amino acid formula (0913131)",
+		"0913131B0 # Powder amino acid semi solid  (0913131)",
+		"0913355A0 # Powder discontinued amino acid formula (0913355)",
+		"0913361A0 # Powder discontinued amino acid formula (0913361)",
+		"0913101A0 # Powder extensively hydrolysed formula (0913101)",
+		"0913101A0 # Powder extensively hydrolysed formula (0913101)",
```
Comment (chrisjwood16) [inline openprescribing/measure_definitions/cmpaperpt.json:27]: Duplicated line

File: openprescribing/measure_definitions/aafpercent.json [hunk 3]
```diff
@@ -1,56 +1,48 @@
...
+		"0913355A0 # Powder discontinued amino acid formula (0913355)",
+		"0913361A0 # Powder discontinued amino acid formula (0913361)",
+		"0913101A0 # Powder extensively hydrolysed formula (0913101)",
+		"0913132A0 # Powder extensively hydrolysed formula (0913132)",
+		"0913351A0 # Powder discontinued extensively hydrolysed formula (0913351)"
+	],
+	"authored_by": "richard.croker@phc.ox.ac.uk",
+	"checked_by": "andrew.brown@phc.ox.ac.uk",
+	"date_reviewed": "2023-03-27",
+	"next_review": "2019-10-01",
```
Comment (chrisjwood16) [inline openprescribing/measure_definitions/aafpercent.json:44]: Needs updating

**Comment by chrisjwood16** [comment]
@richiecroker Looking at gluten free measure - I see GF products have _mostly_ moved to new BNF codes. However looking on OpenCodelists it seems however there are a number of GF items that haven't moved. For example: 
Barkat gluten free multigrain rice bread(0904010H0BYAEAA)

Looking up some of these codes it seems they are marked - Not available - so perhaps not of interest for BSA to change - e.g. https://openprescribing.net/dmd/amp/3666011000001102/

I wonder how this might affect the measure though if these were available and prescribed in the past few years? Not huge numbers for this example but combined it might add up:
https://openprescribing.net/analyse/#org=CCG&numIds=0904010H0BYAEAA&denom=nothing&selectedTab=chart

**Pull request merged**